src/commands/test.ts

```diff
@@ -104,6 +104,11 @@ export class TestCommand implements Command {
       description: "Set job visibility to 'public', 'public restricted', 'share', 'team' or 'private'",
       type: String,
     },
+    {
+      name: 'configFile',
```

Comment:
The fix would be in WCT to specify the correct full name:
https://github.com/Polymer/web-component-tester/blob/master/runner/config.ts#L219

in addition i think the simpleOutput should also be fixed, here and in WCT